### PR TITLE
Update doc URL for Salesforce

### PR DESCRIPTION
**Pull request by duncanjsp**

### CODE REVIEW

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — DISMISSED** [review]

**Comment by kmpoppe** [comment]
@Carlgo11 Travis seems to have gotten stuck in reporting the status here. Can you reset that?

**Comment by Carlgo11** [comment]
Seems like Travis-CI didn't receive the PR data.
@dst87 could you enable edits from maintainers to the PR? It should be on the right hand side of your screen.
Screen Shot 2019-08-24 at 17 08 19

**Comment by duncanjsp** [comment]
@Carlgo11 - it was already checked, so I unchecked it and checked it again. Did that help?

File: _data/communication.yml [hunk 1]
```diff
@@ -211,7 +211,7 @@ websites:
       software: Yes
       email: Yes
       hardware: Yes
-      doc: https://developer.salesforce.com/docs/atlas.en-us.identityImplGuide.meta/identityImplGuide/add_time-based_token.htm
+      doc: https://developer.salesforce.com/docs/atlas.en-us.securityImplGuide.meta/securityImplGuide/add_time-based_token.htm
```
Comment (kmpoppe) [inline _data/communication.yml]:
```suggestion
      doc: https://developer.salesforce.com/docs/atlas.en-us.securityImplGuide.meta/securityImplGuide/add_time-based_token.htm
```

File: _data/communication.yml [hunk 2]
```diff
@@ -211,7 +211,7 @@ websites:
...
```
Comment (kmpoppe) [inline _data/communication.yml]:
```suggestion
    exceptions:
      text: "*Travis Reset. Ignore*"
      
```

**Comment by kmpoppe** [comment]
@dst87 This still doesn't seem to work. Could you please move your change to a new branch within your fork of the project and the open another pull request? This seems to have been lost in obscurity ;-)

Thanks!

// Kai

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The issue is that the commit you did wasn't registered by Travis-CI. (I'll report the bug to them)
We could always do another change so that the next commit gets registered.

Looking at the stored logo for Salesforce it could certainly be improved.
@dst87 could you please merge my pull request over at dst87#1 and that should be enough to get Travis to accept this PR.

**Comment by duncanjsp** [comment]
@Carlgo11 I've merged your PR.  If that doesn't fix it, I'll re-do the PR entirely from a new branch as @kmpoppe suggested.  Let me know!

**Review by Carlgo11 — APPROVED** [review]
It worked.

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**